CONTRIBUTORS.txt

```diff
@@ -112,14 +124,18 @@ needs to be listed here.
 - Tibor Bödecs <mail.tib@gmail.com>
 - Tim Condon <0xtimc@gmail.com>
 - Tim Kientzle <tkientzle@apple.com>
+- Timothée Peignier <timothee.peignier@icloud.com>
 - Tom Counsell <tamc@greenonblack.com>
 - Tomer Doron <tomerd@apple.com>
 - Trevör <adtrevor@users.noreply.github.com>
 - Trevör Anne Denise <adtrevor@users.noreply.github.com>
 - Volodymyr Sapsai <vsapsai@users.noreply.github.com>
 - Will Lisac <will@lisac.org>
 - Wilson Ding <hello@wilsonding.com>
+- buttaface <butta@fastem.com>
 - fadi-botros <botros_fadi@yahoo.com>
+- jemmons <jemmons@users.noreply.github.com>
```

Comment:
@jemmons if you'd like a proper email address (and/or a real name) here, please let me know which.